src/components/ErrorHandling/Sentry.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  ErrorBoundary as SentryErrorBoundary,
+  init,
+  captureException,
+  withScope,
+} from "@sentry/react";
+import type { Scope, SeverityLevel } from "@sentry/react";
+import type { Extras } from "@sentry/types";
+import { environmentVariables } from "utils";
+import ErrorFallback from "./ErrorFallback";
+import { CustomBugsnagError } from "./types";
+
+const { getReleaseStage, getSentryDSN, isProduction } = environmentVariables;
+
+const initializeSentry = () => {
+  const releaseStage = getReleaseStage() || "development";
+  try {
+    init({
+      dsn: getSentryDSN(),
+      debug: !isProduction(),
+      release: APP_VERSION,
+      environment: releaseStage,
+    });
+  } catch (e) {
+    console.error("Failed to initialize Sentry", e);
+  }
+};
+
+const isInitialized = () => {
+  // @ts-expect-error
+  const globalSentry = window.__SENTRY__; // eslint-disable-line no-underscore-dangle
+
+  // If there is a global __SENTRY__ that means that in any of the callbacks init() was already invoked
```

Comment:
I didn't look into this too much but checking if `Sentry.getCurrentHub().getClient()` isn't undefined could potentially be an option here too 😲?

Reply:
did this end up not working? mostly just curious!

Reply:
Simply forgot to commit this 🤦‍♀️